pom.xml

```diff
@@ -83,7 +83,7 @@
     <aws.sdk.version>1.11.133</aws.sdk.version>
     <bigquery.connector.hadoop2.version>0.10.2-hadoop2</bigquery.connector.hadoop2.version>
     <bouncycastle.version>1.56</bouncycastle.version>
-    <cdap.version>6.1.0-SNAPSHOT</cdap.version>
+    <cdap.version>6.0.1-SNAPSHOT</cdap.version>
```

Comment:
how come this is needed?

Reply:
Ah! will change this. When I was going the change on Saturday, the issue was 'develop' builds hadn't passed yet, so I used the release that was available to make progress. I will change this to 6.1.0-SNAPSHOT. Thanks for catching it.